backend/wmg/data/wmg_cube.py

```diff
@@ -49,6 +50,27 @@ def create_cell_count_cube(corpus_path: str):
             .size()
         )
         df = df.rename(columns={"size": "n_cells"})
+
+        mat = np.tile(df.columns[:-1].values[None, :], (df.shape[0], 1)) + "__" + df.iloc[:, :-1].values
+        nodes = np.unique(mat.astype("str"))
+
+        mapper = pd.Series(index=nodes, data=np.arange(nodes.size))
```

Comment:
Initially thought I would be encoding in an adjacency matrix, so I translated strings into indices. But I ended up saving relationships as a hash table, so you don't need this mapper/reverse mapper anymore. Pls remove.